app/controllers/categories_controller.rb

```diff
@@ -5,11 +5,13 @@ class CategoriesController < ApplicationController
   include BreadcrumbBuilder
 
   def index
-    @categories = Category.includes(:translations).roots
+    @categories = Category.includes(:translations, concepts: %i[data_elements])
+                          .roots
```

Comment:
Is this needed? do we have any concepts sitting below the root elements?

Reply:
You're right, we shouldn't have any